src/components/dashboard.tsx

```diff
@@ -122,6 +123,9 @@ export default function Dashboard() {
     stakes: [],
   });
 
+  const [stakingBalance, setStakingBalance] = useState("0.0000");
+  const [rewardsBalance, setRewardsBalance] = useState("0.0000");
```

Comment:
Deprecate these states after we get react query for the queries on staking balance and rewards balances